backend/src/main/java/de/njsm/versusvirus/backend/telegram/InlineButtonCallbackDispatcher.java

```diff
@@ -167,7 +167,7 @@ public void handleWithdrawingHelp(Message message, User user, UUID purchaseId) {
         if (purchase.getStatus() == Purchase.Status.VOLUNTEER_FOUND) {
             purchase.getVolunteerApplications().remove(volunteer.getId());
             if (purchase.getVolunteerApplications().isEmpty()) {
-                purchase.setStatus(Purchase.Status.PUBLISHED);
+                purchase.setStatus(Purchase.Status.NEW);
```

Comment:
Is this to attract more attention from volunteers? This is dangerous and buggy as the telegram chat contains a message with an offer-help button. If a vol clicks that button on a purchase in state new he will get a message stating "the purchase is already assigned" which is wrong. He also won't be listed in the applications, see same file line 80ff

Reply:
@rakshi00 ^